.travis.yml

```diff
@@ -1,12 +1,22 @@
+sudo: false
+
 language: node_js
 
 node_js:
   - "0.10"
 
+addons:
+  apt:
+    sources:
+      - git-core
```

Comment:
I'm pretty sure it can just be `git` by itself and not reference `git-core` anymore.  `git-core` was a transitional name because of goofy reasons — it should now just be a dummy package that points to `git` as its only dependency.